api/changes/ana_4667-fix-unstakes-not-shown-correctly

```diff
@@ -0,0 +1 @@
+[Fixed] [#4668](https://github.com/cosmos/lunie/pull/4668) Fix coinLookup for old balance query @Bitcoinera
```

Comment:
is this correct? this PR feels like handling a lot of other stuff

Reply:
yes, because this is something I also fixed (one line change)